### DISCUSSION TITLE
What are some of the FIL lending facilities for miners?

### DISCUSSION THREAD

**Discussion by momack2** [filecoin-project/community#184] [Mining]
With HyperDrive - there is an increased demand for storage collateral to utilize the increase in network throughput. Miners can get access to FIL to use as storage collateral from these different lending services*:

- Anchorage: https://www.anchorage.com/services/financing
- Coinlist: https://coinlist.co/filecoin-borrower-onboarding
- Darma: https://darma.capital/
- Huobi: https://www.huobi.com/en-us/financial/pledge/

*Add a comment if you'd like to add or remove a name from this list! Please keep the list sorted alphabetically.

**Comment by pk-controller** [comment]
Genesis: https://genesistrading.com/lending/

**Comment by j1v** [comment]
https://www.binance.com/en/loan

**Comment by j1v** [comment]
https://app.aave.com/reserve-overview/RENFIL?pool=AaveV2 -> https://bridge.codefi.network/convert/renfil-filecoin/1

**Reply by filecoinvs** [reply]
> https://app.aave.com/reserve-overview/RENFIL?pool=AaveV2 -> [https://bridge.codefi.network/convert/renfil-filecoin/1](https://filecoinvs.com)

Thanks.

**Comment by TimGuo7** [comment]
Just sharing for the newly updates FIL lend with Coinlist here: 
https://coinlist.co/borrow-fil/?utm_source=newsletter&utm_medium=email&utm_campaign=newsletter_11042021&utm_term=borrow
with fixed-term, started with 3000FIL for 3months.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
darma capital and anchorage offer fielcoin loans too
